joint_trajectory_controller/test/test_trajectory_controller_utils.hpp

```diff
@@ -94,6 +94,10 @@ class TestableJointTrajectoryController
     return last_commanded_state_;
   }
 
+  bool get_has_position_command_interface() { return has_position_command_interface_; }
```

Comment:
```suggestion
  bool has_position_command_interface() { return has_position_command_interface_; }
```
let's improve readability here